### PR TITLE
[Elixir] Move String Literal concept into Basics

### PR DESCRIPTION
**Pull request by jwarwick**
Addresses #2514

### CODE REVIEW

**Review by angelikatyborska — APPROVED** [review]
Looks great to me 👏 but let's also get @neenjaw's approval before merging because I think he had some concerns about this topic.

**Comment by neenjaw** [comment]
I think this is a good iteration, I think makes more sense to move it into basics than get rid of it.  :+1:

**Review by neenjaw — APPROVED** [review]

**Comment by angelikatyborska** [comment]
@jwarwick I just realized we forgot one tiny detail - to mention how to create strings in the introduction! Without that, students won't be able to solve the exercise.

**Pull request merged**
